tests/test_docker.py

```diff
@@ -14,7 +13,7 @@
 
 from .simulationbase import (
     SimulationBase,
-    task_flow_helper,
+    task_flow_helper as _,
```

Comment:
Why does it even need to be imported?

Reply:
You can't use this fixture if it's not imported this way.